hail/src/main/scala/is/hail/expr/types/physical/PType.scala

```diff
@@ -276,7 +276,7 @@ abstract class PType extends Serializable with Requiredness {
 
   def subsetTo(t: Type): PType = {
     // FIXME
-    t.physicalType
+    PType.canonical(t)
```

Comment:
I think this is stacked on the subsetTo pr

Reply:
(And since that has a more complete solution, we can omit this change)

Reply:
not stacked -- they'll just conflict. Whichever goes in first will have to change the other.